src/addons/metaservice/src/test/resources/__files/petstore.yaml

```diff
@@ -5,7 +5,7 @@ info:
   license:
     name: MIT
 servers:
-  - url: http://petstore.swagger.io/v1
+  - url: http://petstore.swagger.io/v1/cats
```

Comment:
It would be good it the test included also negative cases, malformed and empty/null urls
```
- url: null
- url: ["shoud", "be", "a string"]
...
```

Reply:
Added some negative test cases